themes/bootstrap3/templates/RecordTab/holdingsils/electronic.phtml

```diff
@@ -0,0 +1,24 @@
+<h3><?=$this->transEsc('Electronic')?></h3>
+<table class="table table-striped">
+  <?php $i = 0; foreach ($this->holdings as $item): ?>
```

Comment:
It appears that `$i` is unused. Can we remove it?

Reply:
Oops. Yes.